control-plane-operator/controllers/hostedcontrolplane/assets/assets.go

```diff
@@ -0,0 +1,45 @@
+package assets
+
+import "embed"
+
+//go:embed apiserver-haproxy/*
```

Comment:
Is it possible to move this go file up a directory and then //go:embed assets/*/* ? 
(just to not have to keep track of what directories are embedded)

Reply:
Yes, but I think it might require some more tweaks given the current code's references... for this version of the change I was trying to shim with minimal changes. Probably worth a followup

Reply:
👍